python/ray/_raylet.pyx

```diff
@@ -1262,20 +1255,32 @@ async def execute_streaming_generator_async(
             raise
         except Exception as e:
             output_or_exception = e
+            is_exception = 1
 
         loop = asyncio.get_running_loop()
         worker = ray._private.worker.global_worker
-        # Run it in a separate thread to that we can
-        # avoid blocking the event loop when serializing
-        # the output (which has nogil).
-        done = await loop.run_in_executor(
-            worker.core_worker.get_thread_pool_for_async_event_loop(),
-            report_streaming_generator_output,
-            output_or_exception,
-            context)
-        if done:
+
+        futures.append(
```

Comment:
So IIUC, the change here is that you don't wait for each report_streaming_generator_output right?